src/sentry/static/sentry/app/views/issueList/actions.jsx

```diff
@@ -361,18 +411,33 @@ const IssueListActions = createReactClass({
             <Checkbox onChange={this.handleSelectAll} checked={pageSelected} />
           </ActionsCheckbox>
           <ActionSet>
-            <ResolveActions
-              hasRelease={hasReleases}
-              latestRelease={latestRelease}
-              orgId={orgId}
-              projectId={projectId}
-              onUpdate={this.handleUpdate}
-              shouldConfirm={this.shouldConfirm('resolve')}
-              confirmMessage={confirm('resolve', true)}
-              confirmLabel={label('resolve')}
-              disabled={resolveDisabled}
-              disableDropdown={resolveDropdownDisabled}
-            />
+            {selectedProjectSlug ? (
+              <Projects orgId={orgId} slugs={[selectedProjectSlug]}>
+                {({projects, initiallyLoaded, fetchError}) => {
+                  const selectedProject = projects[0];
+                  return fetchError ? (
+                    <LoadingError message={t('Error loading project')} />
```

Comment:
Hmm what does this look like in the bar? We might need this to fail a bit more gracefully.

Reply:
yea it looks real bad actually :( 

![image](https://user-images.githubusercontent.com/9372512/70673398-53077c80-1c37-11ea-8f07-3606e693d711.png)

Not sure if we even need to display a fetchError there. I know in other areas we don't catch errors from the `<Projects>` component. If it fails worst case is that users can't use the resolve drop down.

Reply:
What about rendering the Resolve button disabled with tooltip that there was an error?

Reply:
sounds good, updated pic in description